lib/idp_constants.rb

```diff
@@ -0,0 +1,10 @@
+module Idp
+  module Constants
```

Comment:
can we make the file match autoloading expectations even if it's not autoloaded? `lib/idp/constants.rb` or `IdpConstants` as the module name?

Reply:
yep